packages/components/src/components/tooltip/tooltip.js

```diff
@@ -151,71 +177,183 @@ class Tooltip extends mixin(
     this.tooltip.changeState(
       state,
       Object.assign(detail, { delegatorNode: this.element }),
-      callback
+      () => {
+        // @questions
```

Comment:
`.changeState()` lets you implement DOM changes, etc. (internal implementations) in `._changeState()`. Please see the following for the details:

https://github.com/carbon-design-system/carbon/blob/v10.3.0/packages/components/src/globals/js/mixins/evented-state.js#L91-L102
https://github.com/carbon-design-system/carbon/blob/v10.3.0/packages/components/src/components/modal/modal.js#L87-L140

Above links may answer your question of callback args, too.

Reply:
@elizabethsjudd Could you move this callback block to `_changeState()`? Thanks!